<ui version="4.0" >
 <class>UpdateBranchForm</class>
 <widget class="QDialog" name="UpdateBranchForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Update Bazaar Branch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>This directory is a branch.  Please select what you would like to update</string>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Update source</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QRadioButton" name="but_pull" >
        <property name="text" >
         <string>Pull most recent changes from:</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>13</width>
            <height>98</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QComboBox" name="location" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable" >
               <bool>true</bool>
              </property>
              <item>
               <property name="text" >
                <string>&lt;Parent Branch shown here></string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="location_picker" >
              <property name="text" >
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="but_pull_remember" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="text" >
             <string>Remember this as the new parent branch</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="but_pull_overwrite" >
            <property name="text" >
             <string>Overwrite differences between branches</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="but_update" >
        <property name="text" >
         <string>Update working tree to the latest changes in the branch</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>but_pull</zorder>
     <zorder>but_update</zorder>
     <zorder>but_pull_remember</zorder>
     <zorder>but_pull_overwrite</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>location</zorder>
     <zorder>horizontalSpacer</zorder>
     <zorder>horizontalSpacer</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
